test/apiv2/01-basic.at

```diff
@@ -68,7 +68,7 @@ for i in $(seq 1 10); do
 done
 t1=$SECONDS
 delta_t=$((t1 - t2))
-if [ $delta_t -le 5 ]; then
+if [ $delta_t -le 7 ]; then
     _show_ok 1 "Time for ten /info requests ($delta_t seconds) <= 5s"
```

Comment:
Should this be 7 seconds?

Reply:
Argh. That's exactly why I hate duplication. Fixed, and refactored so it shouldn't happen again the next time we have to bump up the time limit. Thank you for catching that.